test/test-operation.js

```diff
@@ -32,260 +32,638 @@ var helpers = require('./helpers');
 var Sway = helpers.getSway();
 
 describe('Operation', function () {
-  var swaggerApi;
+  context('swaggerApiRelativeRefs', function () {
```

Comment:
I'm using the same setup, VS Code and the Atom One Dark theme.  As for `context` vs. `describe`, I think it's out of habit honestly.  _(`context` could be someone ambiguous as well.)_